testing/python_lib/test_fot.py

```diff
@@ -215,5 +216,40 @@ def test_ports_states(self):
         self._verify_received_device_behaviors(expected_received_device_behaviors)
 
 
+class FotContainerTest(IntegrationTestBase):
+    """Test suite for dynamic config changes"""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.stack_options['fot'] = True
+        self.stack_options['local'] = True
+
+    def tearDown(self):
+        pass
+
+    def test_dhcp_reflection(self):
+        """Test to check DHCP reflection when on test VLAN"""
+        def dhclient_method(container=None):
```

Comment:
Either use positional args or keyword, but try not to mix them. E.g., here you're using a keyword arg (container=), but when you call it you're just using positional i.e. dhclient_metho('forch-faux-1')  -- just need to fix one of the two places to match the other.

Reply:
Done